services/m3aggregator/config/aggregator.go

```diff
@@ -531,50 +550,70 @@ func (c *kvClientConfiguration) NewKVClient(instrumentOpts instrument.Options) (
 	return c.Etcd.NewClient(instrumentOpts)
 }
 
-type shardFnType string
+type hashFnType string
 
-// List of supported sharding function types.
+// List of supported hashing function types.
 const (
-	murmur32ShardFn shardFnType = "murmur32"
+	murmur32HashFn hashFnType = "murmur32"
```

Comment:
nit: perhaps make this public as well since users will be expected to set in the configuration file?

Reply:
This has been moved to the `sharding` package and yes the hashing types are now exported.